lua/auto-session/autocmds.lua

```diff
@@ -229,6 +229,9 @@ local function setup_dirchanged_autocmds(AutoSession)
 
       AutoSession.AutoSaveSession()
       AutoSession.run_cmds "pre_cwd_changed"
+
+      -- Clear the current session, fixes #399
+      vim.v.this_session = ""
```

Comment:
Interesting, for some reason I thought this was a read_only property

Reply:
Yeah, I think I remember that from #250 

Seems like it's writable, even in older versions (at least according to the unit tests)